lightning/backend/local.go

```diff
@@ -1127,8 +1134,23 @@ func isIngestRetryable(resp *sst.IngestResponse, region *split.RegionInfo, meta
 				Leader: newLeader,
 				Region: region.Region,
 			}
-			return true, newRegion, errors.Errorf("not leader: %s", errPb.GetMessage())
+		} else {
+			var err error
+			for i := 0; ; i++ {
```

Comment:
unlimited retry? 🤔

Reply:
yes, I think it's make sense to retry until GetRegion return non-nil value or an error.

This strategy is the same as br at: https://github.com/pingcap/br/blob/5af97a1a579f0f54f61c191081611bd92058d15e/pkg/restore/import.go#L260-L281